lib/ffe.gd

```diff
@@ -322,10 +322,8 @@ DeclareGlobalFunction( "FFEFamily" );
 ##  </Description>
 ##  </ManSection>
 ##
-BIND_GLOBAL( "FAMS_FFE_LARGE", [ [], [] ] );
-if IsHPCGAP then
-    ShareSpecialObj( FAMS_FFE_LARGE );
-fi;
+BIND_GLOBAL( "FAMS_FFE_LARGE", NEW_SORTED_CACHE(false) );
```

Comment:
I've noticed and added a few more places which could use `NEW_SORTED_CACHE`, like this one.